src/index.css

```diff
@@ -1,4 +1,70 @@
 body {
+  background-color: rgba(253, 253, 243, 0.8);
+  margin: 0 auto;
+}
+
+.wrapper {
+  width: auto;
+  max-width: 1250px;
+  margin: 0 auto;
+}
+
+h1,
+p {
+  font-family: didot;
```

Comment:
I think for named fonts like this, you need to put them in quotes, so `font-family: 'Didot';`

Also, is Didot a font that comes with React?? I don't see it being imported